src/Files.App/Data/Items/IListedItem.cs

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) Files Community
+// Licensed under the MIT License.
+
+using Files.App.ViewModels.Properties;
+using Microsoft.UI.Xaml.Media.Imaging;
+using Windows.Storage;
+
+namespace Files.App.Utils
```

Comment:
This interface was automatically generated by VS.
Should all of the properties be in here? 
It might also be better to move the interfaces and classes from `ListedItem.cs` to separate files.

Reply:
Fyi @0x5bfa

Reply:
> Should all of the properties be in here?

All props inside the ListView class should be in here.

> It might also be better to move the interfaces and classes from ListedItem.cs to separate files.

You mean like `IShortcutItem`? Personally I dont but if you like, feel free.

Reply:
> You mean like IShortcutItem? Personally I dont but if you like, feel free.

yes exactly. if you wouldn't do it, we can also leave it as it is